buildtest/cli/__init__.py

```diff
@@ -1295,3 +1331,24 @@ def cdash_menu(subparsers):
     upload.add_argument(
         "-o", "--open", action="store_true", help="Open CDASH report in browser"
     )
+
+
+def add_hidden_parser(subparser, hidden_parser):
+    """This method adds a help message to certain parsers
+
+    Args:
+        subparsers (argparse._SubParsersAction): Subparser object
+        hidden_parser: dictionary
+    """
+    if "command" not in hidden_parser.keys():
+        for command, help_msg in hidden_parser.items():
+            subparser.add_parser(command, help=help_msg)
+        return
```

Comment:
Remember that python  will return `None` by default from method so this `return` statement is redundant